src/helpers/getCssExports.ts

```diff
@@ -85,6 +86,9 @@ export const getCssExports = ({
               if (output === undefined) {
                 throw new Error('No Less output.');
               }
+
+              // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
```

Comment:
Are the types wrong here? Did you encounter a situation where this could be falsey?

Reply:
No, I get an eslint error:
`Require statement not part of import statement. ` [@typescript-eslint/no-var-requires](https://typescript-eslint.io/rules/no-var-requires)